tests/engine_comprehensive_test.py

```
@@ -0,0 +1,308 @@
+            check="message",
+            regex="^(feat|fix):",
+            error="Invalid commit message",
+            suggest="Use conventional format",
+        )
+        validator = CommitMessageValidator(rule)
+        assert validator.rule == rule
+
+    @patch("commit_check.engine.get_commit_info")
+    @patch("commit_check.engine.has_commits")
+    def test_commit_message_validator_with_stdin(
+        self, mock_has_commits, mock_get_commit_info
```

Ruff (0.13.2)

Unused method argument: `mock_get_commit_info`

(ARG002)